tract_querier/query_processor.py

```diff
@@ -735,11 +738,12 @@ def visit_Name(self, node):
             node
         )
 
-    def visit_Str(self, node):
-        return ast.copy_location(
-            ast.Str(s=node.s.lower()),
-            node
-        )
+    def visit_Constant(self, node):
+        if type(node.s) == str:
```

Comment:
this too is usually better as `isinstance(node.s, str)`.  Most probably this logic will need to be mixed with the one in line 555.  Probably calling two methods, one for each type of constant

Reply:
Not sure what's going on here: when debugging locally the change for `EvaluateQueries` seemed to work fine, but this one was either being evaluated to `False` for some cases or not being called at all (cannot remember) and thus the test were failing.

Reply:
If you happen to come up with a fix, it would be very much appreciated.